src/Site.hs

```diff
@@ -87,7 +89,7 @@ main = shakeArgs shakeOptions $ do
 
         let relativeXzs = [ "./" ++ f ++ ".xz" | f <- files ]
 
-        Stdout sums <- cmd ("sha256sum" : relativeXzs) (Cwd "site/files")
```

Comment:
you might add a note, that this is `Linux -> macOS` change

Reply:
This was yes, but it should also work on Linux? At least Ubuntu does have shasum exe

Reply:
TIL! though it's not an exe, it's a perl script :) :+1: